rerun_py/src/python_session.rs

```diff
@@ -271,4 +305,36 @@ impl PythonSession {
             },
         ));
     }
+
+    /// Get a url to an instance of the web-viewer
+    ///
+    /// This may point to app.rerun.io or localhost depending on
+    /// whether `host_assets` was called.
+    pub fn get_app_url(&self) -> String {
+        #[cfg(feature = "web_viewer")]
+        if let Some(hosted_assets) = &self.self_hosted_web_viewer {
+            return format!("http://localhost:{}", hosted_assets.get_port());
+        }
+
+        let short_git_hash = &self.build_info.git_hash[..7];
+        format!("https://app.rerun.io/commit/{short_git_hash}")
+    }
+
+    /// Start a web server to host the run web-asserts
+    ///
+    /// The caller needs to ensure that there is a `tokio` runtime running.
+    #[allow(clippy::unnecessary_wraps)]
+    pub fn self_host_assets(&mut self, _web_port: Option<u16>) -> Result<(), PythonSessionError> {
+        #[cfg(feature = "web_viewer")]
+        {
+            self.self_hosted_web_viewer = _web_port.map(rerun::web_viewer::HostAssets::new);
+
+            Ok(())
```

Comment:
I think we should either return an error if we weren't compiled with `web_viewer` feature, or - preferably - put the `[cfg(feature = "web_viewer")]` around the whole function